cfme/containers/pod.py

```diff
@@ -8,6 +8,12 @@
 
 list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
 
+
+def get_all_pods():
+    sel.force_navigate('containers_pods')
+    ui_pods = [r.name.text for r in list_tbl.rows()]
+    return ui_pods
+
```

Comment:
Look how @pavelzag does it here, please https://github.com/ManageIQ/integration_tests/pull/3367/files and make it consistent between object types...